### PR TITLE
Update text-spacing.html

### PR DESCRIPTION
**Pull request by mbgower**
Added information on ellipses to address https://github.com/w3c/wcag/issues/728

### CODE REVIEW

**Comment by JAWS-test** [comment]
I disagree with this because it undermines the purpose of the SC

**Comment by mbgower** [comment]
@JAWS-test, perhaps you can elaborate in the issue (so your reasoning is capture)?

**Review by patrickhlauke — APPROVED** [review]

File: understanding/21/text-spacing.html [hunk 1]
```diff
@@ -26,6 +26,16 @@ <h4 id="applicability">Applicability</h4>
          </ul>
          <p>For this SC, <a href="https://www.w3.org/TR/html5/scripting-1.html#the-canvas-element">canvas</a> implementations of text are considered to be <a>images of text</a>.</p>
```
Comment (stevefaulkner) [inline understanding/21/text-spacing.html:27]: suggest changing this link to the link it resolves to

File: understanding/21/text-spacing.html [hunk 2]
```diff
@@ -24,8 +24,18 @@ <h4 id="applicability">Applicability</h4>
...
       	</section>
+         <section>
+            <h4 id="ellipses">Use of ellipses</h4>
+            <p>There may be regions of a page where text containers cannot expand due to design constraints (such as a maximum width for the left navigation or table column headers). A common convention if text exceeds its space is to replace truncated text with an ellipsis. Where ellipses appear as a result of modifying text style properties, the page can still meet the Text Spacing requirements, so long as one or more of the following takes place:</p>
+            <ul>
+            <li>a mechanism is provided to reveal the truncated text on the page (for instance, the text appears on focus or on activation)</li>
+            <li>where the ellipsis is part of a link, the truncated text is revealed on the linked page</li>
+            </ul>
+            
+            <p>Where text is not generally truncated but it is when text is spaced, if there is no mechanism to show the truncated text, it fails the SC.</p>
```
Comment (alastc) [inline understanding/21/text-spacing.html:37]:
```suggestion
            <p>Where text is not truncated but it is when text is spaced, if there is no mechanism to show the truncated text, it fails the SC.</p>
```

File: understanding/21/text-spacing.html [hunk 3]
```diff
@@ -24,8 +24,18 @@ <h4 id="applicability">Applicability</h4>
...
-         <p>For this SC, <a href="https://www.w3.org/TR/html5/scripting-1.html#the-canvas-element">canvas</a> implementations of text are considered to be <a>images of text</a>.</p>
+         <p>For this SC, <a href="https://html.spec.whatwg.org/multipage/canvas.html#the-canvas-element">canvas</a> implementations of text are considered to be <a>images of text</a>.</p>
       	</section>
+         <section>
+            <h4 id="ellipses">Use of ellipses</h4>
+            <p>There may be regions of a page where text containers cannot expand due to design constraints (such as a maximum width for the left navigation or table column headers). A common convention if text exceeds its space is to replace truncated text with an ellipsis. Where ellipses appear as a result of modifying text style properties, the page can still meet the Text Spacing requirements, so long as one or more of the following takes place:</p>
+            <ul>
+            <li>a mechanism is provided to reveal the truncated text on the page (for instance, the text appears on focus or on activation)</li>
+            <li>where the ellipsis is part of a link, the truncated text is revealed on the linked page</li>
```
Comment (alastc) [inline understanding/21/text-spacing.html:34]:
```suggestion
            <li>where the ellipsis is part of a section of 
            content which includes a link, the truncated text is revealed on the linked page</li>
```

**Pull request merged**